app/uk/gov/hmrc/softdrinksindustrylevy/connectors/TaxEnrolmentConnector.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.softdrinksindustrylevy.connectors
+
+import javax.inject.Singleton
+
+import play.api.Logger
+import play.api.libs.json.{JsObject, JsString, Json}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.http.logging.Authorization
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.softdrinksindustrylevy.config.WSHttp
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class TaxEnrolmentConnector extends ServicesConfig {
+
+  val callbackUrl: String = getConfString("tax-enrolments.callback", "")
+  val serviceName: String = getConfString("tax-enrolments.serviceName", "")
+
+  val http: WSHttp.type = WSHttp
```

Comment:
can this be inlined if it's just going to be a fixed reference to an Object that can't be overridden?